app/integrations/discord/interactions/request.rb

```diff
@@ -0,0 +1,44 @@
+module Discord
+  module Interactions
+    class Request
+      INTERACTION_TYPE = {
+        PING: 1,
+        APPLICATION_COMMAND: 2,
+        MESSAGE_COMPONENT: 3,
+        APPLICATION_COMMAND_AUTOCOMPLETE: 4,
+        MODAL_SUBMIT: 5
+      }.freeze
```

Comment:
This could also be a class similar to what you did on `User`

```rb
module Discord
  module Resources
    class Interaction
      PING_TYPE = 1
      APPLICATION_COMMAND_TYPE = 2
      MESSAGE_COMPONENT_TYPE = 3
      APPLICATION_COMMAND_AUTOCOMPLETE_TYPE = 4
      MODAL_SUBMIT_TYPE = 5

      def initialize(type:)
        @type = type
      end

      def ping_type?
        type == PING_TYPE
      end

      def application_command_type?
        type == APPLICATION_COMAND_TYPE
      end

      # ...

      private

      attr_reader :type
    end
  end
end
```